lib/tdf3/src/models/encryption-information.ts

```diff
@@ -102,7 +102,9 @@ export class SplitKey {
           : typeof metadata === 'string'
           ? metadata
           : () => {
-              throw new Error();
+              throw new Error(
```

Comment:
base error?

Reply:
Yes this maybe should be invalidFileError, now that I look closer

Reply:
oh wait nevermind this is during `write`. Configuration error probably